airflow/migrations/versions/d38e04c12aa2_add_serialized_dag_table.py

```diff
@@ -0,0 +1,51 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add serialized_dag table
+
+Revision ID: d38e04c12aa2
+Revises: 6e96a59344a4
+Create Date: 2019-08-01 14:39:35.616417
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'd38e04c12aa2'
+down_revision = '6e96a59344a4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Upgrade version."""
+    op.create_table('serialized_dag',  # pylint: disable=no-member
+                    sa.Column('dag_id', sa.String(length=250), nullable=False),
+                    sa.Column('fileloc', sa.String(length=2000), nullable=False),
+                    sa.Column('fileloc_hash', sa.Integer(), nullable=False),
+                    sa.Column('data', sa.Text(), nullable=False),
+                    sa.Column('last_updated', sa.DateTime(), nullable=False),
```

Comment:
We need to add Timezone support here.

![image](https://user-images.githubusercontent.com/8811558/62812450-aa68d500-bafd-11e9-93a7-0d6be6179973.png)

The time shown in this table is not UTC which **might** cause issues as all the other table have UTC

**dag_run** table:

![image](https://user-images.githubusercontent.com/8811558/62812510-f0be3400-bafd-11e9-8aaf-5098593d761f.png)


I will add that.

Reply:
OK. Thanks!!